bin/oneoff/send_facilitator_decision_emails

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+
+require_relative '../../dashboard/config/environment'
+
+ActiveRecord::Base.transaction do
+  applications_emailed = []
+  puts "Sending decision emails for locked declined/waitlisted facilitator 1920 applications..."
+  Pd::Application::Facilitator1920Application.find_each do |application|
+    if application.locked? && Pd::Application::Facilitator1920Application::AUTO_EMAIL_STATUSES.include?(application.status)
+      log = application.sanitize_status_timestamp_change_log
+
+      if log.select {|entry| ['declined_email', 'waitlisted_email'].include? entry[:title]}.empty?
+        application.queue_email(application.status, deliver_now: true)
+        applications_emailed << "id: #{application.id}, status: #{application.status}"
+      end
+    end
+  rescue => e
+    puts "Failure on application #{application.id}:\n#{e.message}\n#{application&.errors&.messages}"
+    raise
+  end
+
+  applications_emailed.each do |app_string|
```

Comment:
Looks like just `puts applications_emailed` would work as well?

Reply:
ha yes it would!